ganymede/models/sources.yml

```diff
@@ -10,4 +10,12 @@ sources:
       - name: retailers
       - name: stores
       - name: statcan_food_prices
+  - name: static
```

Comment:
You need to resolve a merge conflict here. I think you accidentally deleted a line.